game-core/src/test/resources/required-op-codes.txt

```diff
@@ -0,0 +1,134 @@
+3,games.strategy.engine.chat.IChatChannel,speakerAdded,org.triplea.domain.data.ChatParticipant
```

Comment:
The package name of `org.triplea.domain.data.ChatParticipant` was changed by hand